parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/xcm_config.rs

```diff
@@ -334,15 +337,20 @@ impl ExportXcm for BridgeHubRococoOrBridgeHubWococoSwitchExporter {
 				message,
 			)
 			.map(|result| ((Wococo, result.0), result.1)),
+			location if location == EthereumNetwork::get() && relay == NetworkId::Rococo =>
+				Err(Unroutable),
```

Comment:
Is this an obsolete PR? Surely we don't want to error, but rather pass the message to the underlying exporter for out bridge?

Reply:
The convert-xcm PR which merges into this one replaces this part of code with the actual implementation.

https://github.com/Snowfork/cumulus/pull/17
https://github.com/Snowfork/snowbridge/pull/805